common/djangoapps/student/models.py

```diff
@@ -2917,6 +2917,18 @@ class BulkUnenrollConfiguration(ConfigurationModel):
     )
 
 
+class BulkUpdatenrollmentConfiguration(ConfigurationModel):
```

Comment:
Can we keep the name same as the management command `BulkChangeEnrollmentCSVConfig`? Otherwise it might be confusing for some folks.